R-package/tests/testthat/test_lgb.importance.R

```diff
@@ -32,8 +32,8 @@ test_that("lgb.importance() should reject bad inputs", {
         , "lightgbm.model"
     )
     for (input in bad_inputs) {
-        # expect_error({
-        #     lgb.importance(input)
-        # }, regexp = "'model' has to be an object of class lgb\\.Booster")
+        expect_error({
```

Comment:
the fact that this causes an error gave me an idea! I think that this test is causing a problem because the `lgb.Dataset` is an anonymous object. I bet if I named it like `ds <- lgb.Dataset(...)`, then later freed with it, it would be ok.

For this particular test, I'm going to just remove the `lgb.Dataset` so we don't have to worry about it.